lang/csharp/src/apache/ipc/Specific/SpecificRequestor.cs

```diff
@@ -87,8 +87,8 @@ public override object ReadResponse(Schema writer, Schema reader, Decoder decode
 
         public override Exception ReadError(Schema writer, Schema reader, Decoder decoder)
         {
-            var response = new SpecificReader<object>(writer, reader).Read(null, decoder);
-
+            var response = new SpecificReader<object>(writer, reader).Read(null, decoder);
```

Comment:
This seems unrelated too.

Reply:
Strange, When doing a git blame on my machine I only see the copyright url change near the top of the file.